resources/views/layouts/admin.blade.php

```diff
@@ -38,7 +47,13 @@ class="bg-transparent border-b transition-colors duration-300 w-3/4 text-gray-40
 </div>
 @stack('before_main_scripts')
 <script src="{{asset('js/admin.js')}}"></script>
+
+{{--//add jquery--}}
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
jquery and bootstrap is already included.

Reply:
not resolved yet.